src/auth/FormElement.ts

```diff
@@ -70,6 +70,11 @@ export default class GmfAuthForm extends GmfBaseElement {
           this.checkUserMustChangeItsPassword_();
           this.onUserStateUpdate_(user.getState());
         },
+      }),
+      user.getLoginMessage().subscribe({
+        next: (message) => {
+          this.loginInfoMessage = message;
+        },
```

Comment:
line 47 =>
```
@state loginInfoMessage = '';
```
and remove loginInfoMessage in PanelElement...

Reply:
Done.